spec/factories/shopping_cart_items.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+ factory :shopping_cart_item do
+   quantity 1
+   price_cents 25
+   price_currency "SEK"
+ end
```

Comment:
How do you mean? 
My point is that the factory should create an associated object since an item can not exist on its own (without being part of an ShoppingCart). Or am I mistaken? 
Please check the FactoryGirl documentation (I pointed you to the section about associations once before)